ctapipe/image/muon/muon_reco_functions.py

```diff
@@ -193,6 +196,15 @@ def analyze_muon_event(event):
                 muonintensityoutput.event_id = event.dl0.event_id
                 muonintensityoutput.mask = dist_mask
 
+                muonintensityoutput.ring_completeness = ring_completeness(
+                    x[pix_im is not False], y[pix_im is not False],
+                    pix_im[pix_im is not False],
```

Comment:
`pix_im` is an "array with pixel content, usually pe" right? Why is this compared to a boolean here? 
Also: I think `not False` is  `True`.

if you are testing for `!= 0` here maybe this can be said explicitely.

Also:
```python
selection = pix_im is not False
x_selection = x[selection]
y_selection = y[selection]
pix_im_selection = pix_im[selection]
```

I looks to me like (x, y and pix_im) belong together, just as if they would form a unit, maybe it would be nicer to collect these in a structure array, give this array a meaningful name and give also `selection` a meaningful name. 
I just called it `selection` since I have no idea what is going on.

Reply:
x and y are the coordinates of all the pixels in the camera, while pix_im is the masked array of pixels inside a ring. Since the array is masked, I wanted to extract only the non-zero values and used `!=`, but pep8 complained. I changed it to `np.nonzero`

Apparently, `is not False` is not the same as `is True` :D

Reply:
> Apparently, is not False is not the same as is True :D

Well it is only the same, when comparing booleans. The usage for "True"  and "False" implies to the reader (at least I think so) that both operands of the binary `is` are booleans, which was not the case.